tools/flowc/flowc_usage.flow

```diff
@@ -144,9 +145,10 @@ printUsage(config) {
 	prih(help,"      outfolder=<FOLDER>        Save the typed result of all test cases in the given folder");
 	prih(help,"      verify-types=0            Do not verify the assigned types after typechecking is done");
 	prih(help,"");
-	prih(help,"      server-mode=1             Run flowc in a server mode");
+	prih(help,"      http-server-mode=1        Run flowc as an http server");
 	prih(help,"      server-port=<num>         A port, used in a server mode. Default is '10001'");
 	prih(help,"      server-shutdown=1         Pass this parameter in config in order to stop server");
+	prih(help,"      console-server-mode=1     Run flowc as a console server (uses console for interaction with IDE)");
```

Comment:
Oh, I see. Why not just add "server-mode=http" instead?

Reply:
I changed this to:
`server-mode=<http|console> Run flowc as an http server or a console server accordingly to the option value`